neuralmonkey/learning_utils.py

```diff
@@ -81,8 +81,13 @@ def initialize_tf(initial_variables, threads):
 
     """
     log("Initializing the TensorFlow session.")
-    sess = tf.Session(config=tf.ConfigProto(inter_op_parallelism_threads=threads,
-                                            intra_op_parallelism_threads=threads))
+    cfg = tf.ConfigProto()
+    cfg.inter_op_parallelism_threads=threads
+    cfg.intra_op_parallelism_threads=threads
+    cfg.allow_soft_placement = True # needed for multiple GPUs
+    cfg.log_device_placement = True
```

Comment:
Do you use this log of the device placement somehow? If not, I would delete it, if yes I would make it optional. It generates thousands of lines and make tests on Travis fail because the exceeds 4MB.